src/sage/combinat/partition.py

```diff
@@ -5529,6 +5529,43 @@ def specht_module_dimension(self, base_ring=None):
         from sage.combinat.specht_module import specht_module_rank
         return specht_module_rank(self, base_ring)
 
+    def simple_module_dimension(self, base_ring=None):
+        r"""
+        Return the dimension of the simple module corresponding to ``self``.
+
+        This is equal to the dimension of the Specht module over a field
+        of characteristic `0`.
+
```

Comment:
more explanations are required here. Not clear what the second sentence means. May give the impression that the result does not depend on the base ring.

Reply:
I reworked the sentence to clarify it.